.github/workflows/main.yaml

```diff
@@ -16,12 +16,10 @@ jobs:
         os: [ubuntu-latest, macos-latest]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v2
       - name: Install Nix
-        uses: cachix/install-nix-action@v13
+        uses: cachix/install-nix-action@v15
         with:
-          install_url: https://nixos-nix-install-tests.cachix.org/serve/6l6zv3hfyqp402iax9jayd3i5qkmxx1f/install
-          install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
```

Comment:
That's not required anymore: https://github.com/cachix/install-nix-action/commit/46187b354da8be43b69ee117d29793fa8b0ee3ea